integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -578,6 +582,58 @@ void testAdminServiceAnnotationsLabels() {
         "Missing expected label on admin service");
   }
 
+  /** The default replicas at domain resource level is 1. Patching domain1 with new replicas 3.
+   * Verify managed server pods count at cluster resource level are not affected by domain resource
+   * level patching.
+   * Replicas count at cluster resource level is defined as 2.
+   * Scale up the cluster by patching replicas count to 5 at cluster resource level.
+   * Scale down the cluster by patching replicas count to 1 at cluster resource level.
+   * Verify cluster resource level scaling is successful even though replicas count at domain resource
+   * level is specified as 3
+   */
+  @Test
+  @Order(6)
+  @DisplayName("Verify scaling by patching replicas at domain resource and cluster resource level")
+  void testClusterScale() {
+    final int replicaCount = 2;
+    String clusterName = domainUid + "-" + "cluster-1";
+    final String managedServerPrefix = domainUid + "-managed-server";
+
+    //patch replicas at domain resource level to 3
+    boolean scalingSuccess = scaleAllClustersInDomain(domainUid, domainNamespace, 3);
+    assertTrue(scalingSuccess,
+        String.format("Cluster scaling failed for domain %s in namespace %s", domainUid, domainNamespace));
+
+    //managed server pods at cluster resource level should not be affected by domain resource level patching
+    //verify at cluster resource level there are only 2 managed server pods exist and running
+    //verify managedServer3 doesn't exist
+    logger.info("Check dynamic managed server pods are not affected");
+    assertDoesNotThrow(() -> assertTrue(checkClusterReplicaCountMatches(clusterName,
+        domainNamespace, replicaCount)));
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+    //verify managedServer3 doesn't exist
+    checkPodDoesNotExist(managedServerPrefix + 3, domainUid, domainNamespace);
+
+
+    //scaling by patching replicas at cluster resource level should scale up/down
+    //the cluster even the domain resource has replicas defined
+    logger.info("Updating the cluster {0} replica count to 5", clusterName);
+    boolean p1Success = scaleCluster(clusterName, domainNamespace,5);
+    assertTrue(p1Success,
+        String.format("Patching replica to 5 failed for cluster %s in namespace %s",
+            clusterName, domainNamespace));
```

Comment:
add a check for 5 managed servers are up

Reply:
done